html/partials/502.shtml

```diff
@@ -1 +1,3 @@
-<p>An invalid response was received from the upstream service.</p>
+<h2>502 Bad gateway</h2>
+<p>We're having a technical problem.</p>
```

Comment:
We - think:
"We're having a technical problem.
We’ll try and fix this as soon as possible."